shell/common/application_info.cc

```diff
@@ -47,4 +49,11 @@ std::string GetApplicationUserAgent() {
   return content::BuildUserAgentFromProduct(user_agent);
 }
 
+bool IsAppRTL() {
```

Comment:
Could we combine this with https://github.com/electron/electron/blob/89117fdd999783091e019f7c6c06e7b5a45244d9/shell/browser/browser_mac.mm#L43-L48?

Reply:
Its already combined above https://github.com/electron/electron/pull/39734/files#diff-43f417c3f78558ea55277f32e1a078b6bb6c19be7699a2cf61c7020d78f1110dL43, am I missing something ?

Reply:
nope, i think i misread the commit!